website/docs/plugins/rq_launcher.md

```diff
@@ -0,0 +1,130 @@
+---
+id: rq_launcher
+title: RQ Launcher plugin
+sidebar_label: RQ Launcher plugin
+---
+[![PyPI](https://img.shields.io/pypi/v/hydra-rq-launcher)](https://pypi.org/project/hydra-rq-launcher/)
+![PyPI - License](https://img.shields.io/pypi/l/hydra-rq-launcher)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/hydra-rq-launcher)
+[![PyPI - Downloads](https://img.shields.io/pypi/dm/hydra-rq-launcher.svg)](https://pypistats.org/packages/hydra-rq-launcher)
+[![Example application](https://img.shields.io/badge/-Example%20application-informational)](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_rq_launcher/example)
+[![Plugin source](https://img.shields.io/badge/-Plugin%20source-informational)](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_rq_launcher)
+
+The RQ Launcher plugin provides a launcher for distributed execution and job queuing based on [Redis Queue (RQ)](https://python-rq.org).
+
+### Installation
+This plugin requires Hydra 1.0 (Release candidate)
+```commandline
+$ pip install hydra-rq-launcher --pre
+```
+Usage of this plugin requires a [Redis server](https://redis.io/topics/quickstart).
+
+Note that RQ does [not support Windows](https://python-rq.org/docs/#limitations).
+
+### Usage
+Once installed, add `hydra/launcher=rq` to your command line. Alternatively, override `hydra/launcher` in your config:
+
+```yaml
+defaults:
+  - hydra/launcher: rq
+```
+
+The default configuration packaged with the plugin is:
```

Comment:
I am starting to change how I document the configs for plugins.

Let's try it first in this plugin:
1. remove dataclasses and provide a link to the config class in the repo instead.
2. include the output of:

```
python foo.py --cfg hydra --pacakge hydra.launcher
```

in the doc. try it out. I think it's better to use that.